examples/example.py

```diff
@@ -66,7 +66,7 @@ def subscriber(server="127.0.0.1:7111", timecode=False):
 
             if msg is not None:
                 if msg.msg_name == "USER_MESSAGE":
-                    print(msg)
+                    msg.hexdump()
```

Comment:
I just ran the example and I get output like this:

> 00000000  d2 04 00 00 38 00 00 00  13 4d 82 79 54 85 d8 41 |    8    M yT  A|
00000010  13 4d 82 79 54 85 d8 41  00 00 79 00 00 00 00 00 | M yT  A  y     |
00000020  38 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00 |8               |
00000030  48 65 6c 6c 6f 20 57 6f  72 6c 64 00 00 00 00 00 |Hello World     |
00000040  77 be 9f 1a 2f dd 5e 40  00 00 00 00 01 00 00 00 |w   / ^@        |
00000050  02 00 00 00 03 00 00 00  04 00 00 00 05 00 00 00 |                |
00000060  06 00 00 00 07 00 00 00                          |                |

which is fine for the example but probably not what we want as the default print behavior, but it would still be good to add \_\_str__ and/or \_\_repr__ methods.

Reply:
I'm working on a \_\_repr__ method for the message class. I have it partially working but I may have some questions about ctypes tomorrow. I also realized we should probably update the example to show how to access the data values in the received message.

Reply:
I just committed a __repr__ method that will print the full message structure including fieldnames, data types, and data values. I updated the example to use the new print method (in addition to hexdump, because my print does not convert bytes to strings).